bindings/cpu_profiler.cc

```diff
@@ -134,57 +133,100 @@ v8::Local<v8::Object> CreateSample(uint32_t stack_id, uint32_t sample_timestamp_
   return js_node;
 };
 
+std::string hashCpuProfilerNodeByPath(const v8::CpuProfileNode* node) {
```

Comment:
We are choosing to trade off memory usage vs CPU usage here by concat strings. We could explore options like creating some order preserving hash function, but starting with keeping this simple for now.